common/src/main/java/org/jboss/hal/testsuite/fragment/WindowFragment.java

```diff
@@ -70,18 +70,21 @@ public void waitUntilClosed() {
      * Closes wizard by clicking on cross in top right corner
      */
     public void close() {
-        By selector = By.className(PropUtils.get("modals.window.close.class"));
-        WebElement close = root.findElement(selector);
-
         int windowCount = Console.withBrowser(browser).getWindowCount();
 
-        close.click();
+        clickCross();
 
         waitUntilClosed(windowCount, true);
 
         closed = true;
     }
 
+    protected void clickCross() {
```

Comment:
This was intended since there is no "Close" button, but only the cross in top right corner. If you think it should be better to have `clickClose()`, I can fix it.